x/datadeal/keeper/deal.go

```diff
@@ -77,3 +77,25 @@ func (k Keeper) SetDataSale(ctx sdk.Context, dataSale *types.DataSale) error {
 
 	return nil
 }
+
+func (k Keeper) GetAllDataSaleList(ctx sdk.Context) ([]types.DataSale, error) {
```

Comment:
I wish there was a test for this `get` function.

Reply:
Add test code in [ccf0429](https://github.com/medibloc/panacea-core/pull/426/commits/ccf0429ae930c9882a51c86d89cfacc010be424c).